src/Streamly/Internal/Prelude.hs

```diff
@@ -713,6 +715,11 @@ unfoldrMZipSerial = Serial.unfoldrM
 unfold :: (IsStream t, Monad m) => Unfold m a b -> a -> t m b
 unfold unf x = fromStreamD $ D.unfold unf x
 
+-- | Convert an 'Unfold' with a closed input end into a stream.
```

Comment:
Append an `Internal` annotation until we release it:

```
--
-- /Internal/
```

Reply:
Thanks; the annotation is added.

By the way, I noticed how one of the checks reported a reduced test coverage. I wonder where to put a test if one is preferred. (My guess is `test/Streamly/Test/Array.hs`, but I thought I’d ask first.)

Reply:
It will be great if you can add a test for unfold. Currently there is no test for it. It would go in `Streamly/Test/Internal/Prelude.hs`. We are trying to arrange test code with a 1-to-1 module correspondence with the source modules, but its in flux as of now. As of now many Prelude tests can also be found in test/Prop.hs and test/Main.hs.

BTW, do not worry of the parser test failure on travis, that's a known problem being fixed.

Reply:
Tests added. Coveralls is reporting `unfold0` as “uncovered.” I guess this is due to optimizations.